Extension/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * Add `quoteArgs` to `launch.json` schema. [PR #5639](https://github.com/microsoft/vscode-cpptools/pull/5639)
 * Add logs for a resolved `launch.json` if "engineLogging" is enabled. [PR #5644](https://github.com/microsoft/vscode-cpptools/pull/5644)
 * Add threadExit and processExit logging flags for 'cppvsdbg'. [PR #5652](https://github.com/microsoft/vscode-cpptools/pull/5652)
+* Add support to run cpp build tasks  (Tasks: Run Build Task) [#3674](https://github.com/microsoft/vscode-cpptools/issues/3674)
```

Comment:
This is moved to a 0.30.0-insiders changelog...we could update the changelog in a separate PR, since it's still in flux.